treetime/clock_tree.py

```diff
@@ -203,7 +203,7 @@ def _set_precision_fft(self, precision_fft, precision_branch='auto'):
                         " fft_grid_points=%.3e"%(precision_fft), 2)
                 self.fft_grid_size = precision_fft
             elif precision_fft!='auto':
-                raise ValueError(f"ClockTree: precision_fft needs to be either 'auto' or an integer, got '{precision_fft}'.")
+                raise UnknownMethodError(f"ClockTree: precision_fft needs to be either 'auto' or an integer, got '{precision_fft}'.")
```

Comment:
This should be a new type of error? Something like: Wrong parameter passed - not unknown method. This has nothing to do with a method?